README.md

```diff
@@ -76,10 +74,9 @@ g.start()
 
  - `options` &lt;Object&gt; Options for the link
     - `host` &lt;String&gt; IP to bind to. If null, Grape binds to all interfaces
-    - `dht_maxTables` &lt;Number&gt; Maximum number of DHT tables
+    - `dht_ephemeral` &lt;Boolean&gt; Whether to join the DHT (false) or just query it (true). Default is false
```

Comment:
current defautl behaviour is to join the dht, no?

Reply:
yes - got the bools the wrong way, `ephemeral: false` is query only, not joining

Reply:
yes - `dht_ephemeral: false` means join the DHT - as in "this node is not temporary", default is false